SERVER/templates/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
The HTML files themselves? Not to my knowledge, though Flask isn't something I've explored a whole bunch. I know they include Jinja2 templates, so you can do stuff like 
```
<!doctype html>
<title>Hello from Flask</title>
{% if name %}
  <h1>Hello {{ name }}!</h1>
{% else %}
  <h1>Hello, World!</h1>
{% endif %}`
```
but I'm not sure that's what you're asking.